docs/usage/system_backend/namespace.rst

```diff
@@ -0,0 +1,76 @@
+Namespace
+=====
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Create Namespace
+----------------
+
+.. automethod:: hvac.api.system_backend.namespace.create_namespace
+   :noindex:
+
+Examples
+````````
+
+.. testcode:: sys_mount
```

Comment:
This `sys_mount` label should most likely be updated. You can think of those doctest labels as unittest test classes. I.e., to avoid negative interactions between unrelated test cases, you want to keep them segmented in most all cases.